third_party/2and3/redis/client.pyi

```diff
@@ -117,7 +141,15 @@ class StrictRedis:
     def rename(self, src, dst): ...
     def renamenx(self, src, dst): ...
     def restore(self, name, ttl, value): ...
-    def set(self, name, value, ex=..., px=..., nx=..., xx=...): ...
+    def set(
+        self,
+        name: Text,
```

Comment:
I think `name` (here and in the other methods too) can be either `str` or `bytes`.